src/Nancy.Authentication.Forms.Tests/FormsAuthenticationFixture.cs

```diff
@@ -162,7 +161,71 @@ public void Should_return_ok_response_when_user_logs_in_without_redirect()
             result.StatusCode.ShouldEqual(HttpStatusCode.OK);
         }
 
-        [Fact]
+		#region Throw helpful exception when the configuration is not enabled
+
+		[Fact]
+		public void Should_throw_helpful_exception_message_when_user_logs_in_without_redirect_and_forms_authentication_not_enabled()
+		{
+			// Given
+			const string expectedMessage = "The internal FormsAuthenticationConfiguration has not been set. Ensure that FormsAuthentication has been enabled in the bootstrapper";
+			FormsAuthentication.Disable();

```

Comment:
I don't think it's neccesairy to disable the FormsAuthentication if you don't enable it it's disabled.